### ISSUE TITLE
Add ignore_index option to Accuracy metric

### ISSUE THREAD

**Issue by stas-sl** [PyTorchLightning/metrics#61]
## 🚀 Feature

It would be nice if `Accuracy` metric accepted `ignore_index` as it works for `StatsScore` or `torch.nn.CrossEntropyLoss`

**Comment by avinashsai** [comment]
Hi,
Anyone working on this? I can try the implementation.

**Comment by SkafteNicki** [comment]
@avinashsai please feel free to do so :]
Ping me either here or on slack if you need any help.

**Comment by avinashsai** [comment]
@SkafteNicki I have a query:
This ignore_index should ignore that class label and calculate accuracy for the rest?
Ex: y_pred - [1, 0, 1, 1, 0], y_true - [1, 0, 0, 1, 0] 

Accuracy(y_pred, y_true, ignore_index=0) = 100%
Ignoring class label '0', there are only two samples belonging to class '1' and hence accuracy is 100%.

**Comment by SkafteNicki** [comment]
@avinashsai that at least how I would like the argument to work, as it is similar to how `torch.nn.CrossEntropyLoss` works.

**Comment by SkafteNicki** [comment]
@avinashsai how is it going here?

**Comment by avinashsai** [comment]
@SkafteNicki I am working on it. Will submit PR in a couple of days.

**Comment by SkafteNicki** [comment]
Closed by #166
